tools/viewer/README.md

```diff
@@ -64,25 +64,25 @@ slint-viewer --on open-url 'xdg-open $1' myfile.slint
 
 Be carefull to use single quote or to escape the `$` so that the shell don't expand the `$1`
 
-
 ## Dialogs
 
 If the root element of the .slint file is a `Dialog`, the different StandardButton might close
 the dialog if no callback was set on the button.
 
- - `ok`, `yes`, or `close` buttons accepts the dialog
- - `cancel`, `no` buttons reject the dialog
+- `ok`, `yes`, or `close` buttons accepts the dialog
+- `cancel`, `no` buttons reject the dialog
 
 ## Result code
 
 The program returns with the following error code:
- - If the command line argument parsing fails, the exit code will be *1*
- - If the .slint compilation fails, the compilation error will be printed to stderr and the exit code
-   will be *-1*
- - If a Window is closed, the exit code will be *0*
- - If a Dialog is closed with the "Ok" or "Closed" or "Yes" button, the exit code will be *0*
- - If a Dialog is closed with the "Cancel" or "No" button, or using the close button in the window
-   title bar, the exit code will be *1*
+
+- If the command line argument parsing fails, the exit code will be _1_
```

Comment:
Please reconfigure your reformatting script to ignore on markdown files (`*.md`)

Reply:
Sorry for that: My upstream vim setup turned on format-on-save and I had not noticed it:-/

I have now forced that to off for everything.